amd/src/mod_hsuforum_loader.js

```diff
@@ -42,28 +43,59 @@ define(['jquery'], function ($) {
             posts.push(footerReply);
         }
 
+        var cookie = "Reply=yes";
         if (posts) {
             $(posts).each(function(){
                 const postObserver = new MutationObserver(() => {
                     let form = $(this).find('form');
-
                     if (form) {
                         let formTextarea = $(form).find('.hsuforum-textarea');
                         $(form).on('submit', () => {
                             // Check for form errors
                             if ($(formTextarea).text() != 0) {
                                 document.body.dispatchEvent(spinnerStartEvent);
+                                var postid = $('input:hidden[name=reply]').val(); //forms current hidden reply value, replyto
```

Comment:
How are you targeting the correct postId here? There can be two instances of the hidden reply, one for the footer reply and then one for a thread reply

Reply:
targets the hidden input for the specific form being submitted